src/seedu/addressbook/common/Utils.java

```diff
@@ -35,4 +37,20 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+    
+    /**
+     * Concatenates a list of printables into a single String
+     * @param printables list of items to be printed
+     * @return
+     */
+    public static String getPrintableString(Printable... printables) {
```

Comment:
You should consider if this method is likely to be usable by any other classes. If the method is only going to be used in a specific place, it may not be best to put it here.